butterknife-gradle-plugin/src/main/java/butterknife/plugin/FinalRClassBuilder.java

```diff
@@ -26,7 +26,8 @@
 public final class FinalRClassBuilder {
   private static final String SUPPORT_ANNOTATION_PACKAGE = "android.support.annotation";
   private static final String[] SUPPORTED_TYPES = {
```

Comment:
I saw that but I prefer keeping the supported list. It makes it clear what needs to be tested.